src/commands/synthetics/reporters/default.ts

```diff
@@ -355,25 +368,23 @@ export class DefaultReporter implements MainReporter {
         // Test is either skipped from datadog-ci config or from test config
         const isSkippedByCIConfig = config.executionRule === ExecutionRule.SKIPPED
         if (isSkippedByCIConfig) {
-          return `>> Skipped test "${chalk.yellow.dim(test.name)}"`
+          return `Skipped test "${chalk.yellow.dim(test.name)}"`
         } else {
-          return `>> Skipped test "${chalk.yellow.dim(test.name)}" because of execution rule configuration in Datadog`
+          return `Skipped test "${chalk.yellow.dim(test.name)}" because of execution rule configuration in Datadog`
         }
       }
 
       if (executionRule === ExecutionRule.NON_BLOCKING) {
-        return `Trigger test "${chalk.green.bold(test.name)}" (non-blocking)`
+        return `Found test "${chalk.green.bold(test.name)}" (non-blocking)`
       }
 
-      return `Trigger test "${chalk.green.bold(test.name)}"`
+      return `Found test "${chalk.green.bold(test.name)}"`
     }
 
     this.write(`${idDisplay} ${getMessage()}\n`)
   }
 
   public testWait(test: Test) {
-    const idDisplay = `[${chalk.bold.dim(test.public_id)}]`
-
-    this.write(`${idDisplay} Waiting results for "${chalk.green.bold(test.name)}"\n`)
+    return
```

Comment:
Is `testWait` not used anymore (and `testsWait` is used as a replacement)? If so, should we add a JS doc with a `@deprecated` flag?

Reply:
It is not used in our default reporter, but still available for normal use, so no need to mark it as deprecated